docs/source/reference/pysupported.rst

```diff
@@ -229,6 +229,52 @@ The following functions from the :mod:`operator` module are supported:
 * :func:`operator.truediv`
 * :func:`operator.xor`
 
+.. _pysupported-random:
+
+``random``
+----------
+
+Numba supports top-level functions from the :mod:`random` module, but does
+not allow you to create individual Random instances.  A Mersenne-Twister
+generator is used, with a dedicated internal state.  It is initialized at
+startup with entropy drawn from the operating system.
+
+* :func:`random.betavariate`
+* :func:`random.expovariate`
+* :func:`random.gammavariate`
+* :func:`random.gauss`
+* :func:`random.getrandbits`: number of bits must not be greater than 64
```

Comment:
It may be useful to do a pass over all nopython-supported features and document which ones are CUDA-supported. Many things seem injected in the CPU context (see the CPUContext constructor).